From 94035a1ef1b9dbc563e23966f168f873c863b31c Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Thu, 28 Aug 2014 14:23:43 -0400
Subject: [cpufreq] Fix checkpatch errors and warnings

Message-id: <1409235879-1283-130-git-send-email-prarit@redhat.com>
Patchwork-id: 89429
O-Subject: [RHEL7.1 PATCH BZ 1134639 129/185] cpufreq: Fix checkpatch errors and warnings
Bugzilla: 1134369
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Mark Langsdorf <mlangsdo@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1134639

commit e5c87b762871c3377e456f6a57769dc63bdddd4b
Author: Stratos Karafotis <stratosk@semaphore.gr>
Date:   Wed Mar 19 23:29:17 2014 +0200

    cpufreq: Fix checkpatch errors and warnings

    Fix 2 checkpatch errors about using assignment in if condition,
    1 checkpatch error about a required space after comma
    and 3 warnings about line over 80 characters.

    Signed-off-by: Stratos Karafotis <stratosk@semaphore.gr>
    Acked-by: Viresh Kumar <viresh.kumar@linaro.org>
    Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>

Cc: Lenny Szubowicz <lszubowi@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/cpufreq/cpufreq.c b/drivers/cpufreq/cpufreq.c
index c8a1150..fd86af3 100644
--- a/drivers/cpufreq/cpufreq.c
+++ b/drivers/cpufreq/cpufreq.c
@@ -939,8 +939,11 @@ static int cpufreq_add_policy_cpu(struct cpufreq_policy *policy,
  up_write(&policy->rwsem);
 
  if (has_target()) {
-  if ((ret = __cpufreq_governor(policy, CPUFREQ_GOV_START)) ||
-   (ret = __cpufreq_governor(policy, CPUFREQ_GOV_LIMITS))) {
+  ret = __cpufreq_governor(policy, CPUFREQ_GOV_START);
+  if (!ret)
+   ret = __cpufreq_governor(policy, CPUFREQ_GOV_LIMITS);
+
+  if (ret) {
    pr_err("%s: Failed to start governor\n", __func__);
    return ret;
   }
@@ -1393,10 +1396,12 @@ static int __cpufreq_remove_dev_finish(struct device *dev,
   if (!cpufreq_suspended)
    cpufreq_policy_free(policy);
  } else if (has_target()) {
-  if ((ret = __cpufreq_governor(policy, CPUFREQ_GOV_START)) ||
-    (ret = __cpufreq_governor(policy, CPUFREQ_GOV_LIMITS))) {
-   pr_err("%s: Failed to start governor\n",
-          __func__);
+  ret = __cpufreq_governor(policy, CPUFREQ_GOV_START);
+  if (!ret)
+   ret = __cpufreq_governor(policy, CPUFREQ_GOV_LIMITS);
+
+  if (ret) {
+   pr_err("%s: Failed to start governor\n", __func__);
    return ret;
   }
  }
@@ -2083,7 +2088,7 @@ static int cpufreq_set_policy(struct cpufreq_policy *policy,
  if (old_gov) {
   __cpufreq_governor(policy, CPUFREQ_GOV_STOP);
   up_write(&policy->rwsem);
-  __cpufreq_governor(policy,CPUFREQ_GOV_POLICY_EXIT);
+  __cpufreq_governor(policy, CPUFREQ_GOV_POLICY_EXIT);
   down_write(&policy->rwsem);
  }
 
-- 
1.7.1